src/ansys/additive/core/simulation_task_manager.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (C) 2023 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""Manages simulation tasks."""
+
+from __future__ import annotations
+
+from ansys.additive.core.progress_handler import IProgressHandler, Progress
+from ansys.additive.core.simulation_task import SimulationTask
+
+
+class SimulationTaskManager:
+    """Provides a manager for simulation tasks."""
+
+    def __init__(self):
+        """Initialize the simulation task manager."""
+        self._tasks: list[SimulationTask] = []
+
+    def add_task(self, task: SimulationTask):
+        """Add a task to this manager.
+
+        Parameters
+        ----------
+        task: SimulationTask
+            The simulation task holding the long running operation and corresponding server.
```

Comment:
```suggestion
            The simulation task holding the long-running operation and corresponding server.
```

Reply:
Leaving as is. Google API spells it this way.

Reply:
Fixed.